### PR TITLE
The etc and dot is seprated

### PR DESCRIPTION
**Pull request by yu8833**
Signed-off-by: YuPengZTE yu.peng36@zte.com.cn

### CODE REVIEW

File: docs/installation/linux/gentoolinux.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ IRC channel on the Freenode network.
 | btrfs         |         |Enables dependencies for the "btrfs" graph driver, including necessary kernel flags.|
 | contrib       |  Yes    |Install additional contributed scripts and components.|
 | device-mapper |  Yes    |Enables dependencies for the "devicemapper" graph driver, including necessary kernel flags.|
-| doc           |         |Add extra documentation (API, Javadoc, etc). It is recommended to enable per package instead of globally.|
+| doc           |         |Add extra documentation (API, Javadoc, etc.) It is recommended to enable per package instead of globally.|
```
Comment (cpuguy83) [inline docs/installation/linux/gentoolinux.md]: I think the dot is the period for the sentence.

File: docs/security/security.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ Finally, if you run Docker on a server, it is recommended to run
 exclusively Docker in the server, and move all other services within
 containers controlled by Docker. Of course, it is fine to keep your
 favorite admin tools (probably at least an SSH server), as well as
-existing monitoring/supervision processes (e.g., NRPE, collectd, etc).
+existing monitoring/supervision processes (e.g., NRPE, collectd, etc.)
```
Comment (cpuguy83) [inline docs/security/security.md]: I think the dot is the period for the sentence.

File: docs/installation/linux/gentoolinux.md (hunk 1 above)
Comment (mstanleyjones) [inline docs/installation/linux/gentoolinux.md]: I suggest rewording this to something like "Add extra documentation, such as API and Javadoc."

File: docs/security/security.md [hunk 3]
```diff
@@ -136,7 +136,7 @@ Finally, if you run Docker on a server, it is recommended to run
 exclusively Docker in the server, and move all other services within
 containers controlled by Docker. Of course, it is fine to keep your
 favorite admin tools (probably at least an SSH server), as well as
-existing monitoring/supervision processes (e.g., NRPE, collectd, etc).
+existing monitoring/supervision processes, such as NRPE and collectd.
```
Comment (YuPengZTE) [inline docs/security/security.md]: Thanks @cpuguy83, thanks @mstanleyjones. Here the "e.g." and "etc." are repeat, and I think the words of mstanleyjones are appropriate. When there is ambiguity, we can use another expression.

File: docs/userguide/eng-image/dockerfile_best-practices.md [hunk 4]
```diff
@@ -243,7 +243,7 @@ service (Apache, Rails, etc.), you would run something like
 recommended for any service-based image.
 
 In most other cases, `CMD` should be given an interactive shell (bash, python,
-perl, etc), for example, `CMD ["perl", "-de0"]`, `CMD ["python"]`, or
+perl, etc.), for example, `CMD ["perl", "-de0"]`, `CMD ["python"]`, or
```
Comment (mstanleyjones) [inline docs/userguide/eng-image/dockerfile_best-practices.md]: I think you could rewrite this in a similar way to the others.

Comment (YuPengZTE) [inline docs/userguide/eng-image/dockerfile_best-practices.md]: Thanks @mstanleyjones. When I change it to "such as bash, python 
and perl, for example...", I find it is ambiguity, so add dot before "For example".

Comment (mstanleyjones) [inline docs/userguide/eng-image/dockerfile_best-practices.md]: Why not say something like:

In most other cases, `CMD` should be an interactive shell, such as Bash, Python, or Perl. The following examples use Perl and Python:
    ```
    CMD ["perl", "-de0"]
    ```
    ```
    CMD ["python"]
    ```

File: docs/userguide/eng-image/dockerfile_best-practices.md [hunk 5]
```diff
@@ -238,12 +238,12 @@ keep the image size down. Since the `RUN` statement starts with
...
 image, along with any arguments. `CMD` should almost always be used in the
 form of `CMD [“executable”, “param1”, “param2”…]`. Thus, if the image is for a
-service (Apache, Rails, etc.), you would run something like
+service, such as Apache and Rails, you would run something like
 `CMD ["apache2","-DFOREGROUND"]`. Indeed, this form of the instruction is
 recommended for any service-based image.
 
-In most other cases, `CMD` should be given an interactive shell (bash, python,
-perl, etc), for example, `CMD ["perl", "-de0"]`, `CMD ["python"]`, or
+In most other cases, `CMD` should be given an interactive shell, such as bash, python
```
Comment (sfsmithcha) [inline docs/userguide/eng-image/dockerfile_best-practices.md]: nit: comma after python

**Comment by sfsmithcha** [comment]
@YuPengZTE , please sign your PR.

File: docs/userguide/eng-image/dockerfile_best-practices.md (hunk 4 above)
Comment (mstanleyjones) [inline docs/userguide/eng-image/dockerfile_best-practices.md]: @YuPengZTE please address this comment (it got hidden somehow) and sign your commit using the -s option when you amend the commit. Let me know if you have more questions about this. Thank you so much for your submission.

**Comment by vdemeester** [comment]
@sfsmithcha the commit is signed :wink:

**Comment by vdemeester** [comment]
… and LGTM 🐸

**Comment by sfsmithcha** [comment]
thanks @vdemeester , sorry for the confusion @YuPengZTE

**Comment by thaJeztah** [comment]
LGTM
